src/blockdiagonal.jl

```diff
@@ -39,7 +39,7 @@ Return the size of each on-diagonal block in order.
 julia> B = BlockDiagonal([rand(2, 2), rand(3, 3)]);
 
 julia> blocksizes(B)
-2-element Array{Tuple{Int64,Int64},1}:
+2-element Vector{Tuple{Int64, Int64}}:
```

Comment:
I decided it was easy enough to update this to the 1.6 rather than pinning docs to build in 1.5.
Since this was the only change that was needed